images/Makefile.common.in

```diff
@@ -13,7 +13,7 @@ IMAGE?=$(REGISTRY)/$(IMAGE_NAME):$(TAG)
 export DOCKER_CLI_EXPERIMENTAL=enabled
 
 # build with buildx
-PLATFORMS?=linux/amd64,linux/arm64,linux/ppc64le
+PLATFORMS?=linux/amd64,linux/arm64,linux/ppc64le,linux/mips64le
```

Comment:
you should try building the images, particularly `make -C images/base build`, i'm quite certain this will fail in it's current state, until the dependencies are sorted, but should check the others too.

this is not run in CI, as these images are modified relatively infrequently at the moment, and manually pre-build/pushed and tested prior to merge currently, but you can build it locally.

Reply:
The dependency of the local kind has been resolved, and we need to push related projects and images to the community. Please continue to support us in the follow-up and help us build mips64le ecology in the cloud native field.

Reply:
I'm not sure I'm quite understanding this comment, but since this PR won't build locally for me
/hold
for now.